java/practice/apiPractice/src/main/java/model/DogCompareResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.springDemo.apiPractice.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DogCompareResponse {
+    String oldestDogName;
```

Comment:
cam both of these fields be set to private?

Reply:
yes and done